client/state/selectors/get-request.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { getRequest, getRequestKey } from 'state/data-layer/wpcom-http/utils';
+
+/**
+ * Returns meta information about data requests going through the data layer
+ *
+ * @param {Object} state Redux state
+ * @param {Object} action data request action
+ * @returns {*} metadata about request
+ */
+export default ( state, action ) => {
+	const data = getRequest( state, getRequestKey( action ) );
```

Comment:
We deleted `getRequest` in utils, so this will be 😢

Reply:
fixed in 0f1f269 - thanks!